### DISCUSSION TITLE
Saving the Panorama

### DISCUSSION THREAD

**Discussion by rjbcollege** [OpenStitching/stitching#204] [General]
I have searched, honestly I have, but I cannot find, nor believe it doesn't exist, anywhere where it says how to save the stitched panorama to a file.
Surely that is the entire point of using stitch, to stitch together images into a larger image - so it needs to be saved.
It's not even mentioned in the tutorial.  That talks about using plot to display it, but not about saving.
It's not a subject of any previous discussion that the word save could find (there are discussions but not matching this question).
I must be missing something, it must be me not reading something somewhere, but please, how do I save my resulting panorama to a file?

**Comment by lukasalexanderweber** [comment]
You should use opencv's `imwrite`, you can see how it's done in the command line interface:

https://github.com/OpenStitching/stitching/blob/cf74fdf610ff655968ddc23bd46033175ec234a0/stitching/cli/stitch.py#L313
